src/test/java/de/rabea/game/GameTest.java

```diff
@@ -19,9 +19,9 @@ public void setup() {
     }
 
     @Test
-    public void playsHuman3x3GameOnce() {
-        FakeHumanPlayer fakePlayer1 = new FakeHumanPlayer(fakeConsoleUI, X);
-        FakeHumanPlayer fakePlayer2= new FakeHumanPlayer(fakeConsoleUI, O);
+    public void playsHumanGameOnce() {
+        FakeConsolePlayer fakePlayer1 = new FakeConsolePlayer(fakeConsoleUI, X);
+        FakeConsolePlayer fakePlayer2= new FakeConsolePlayer(fakeConsoleUI, O);
         fakePlayer1.futureMoves(0,1,2);
         fakePlayer2.futureMoves(4,5);
```

Comment:
Hiding it in a method is a band aid.
Maybe you can work on making the numbers more descriptive?
Or the method name? And with "more descriptive" I don't mean longer, but just more precise.
Maybe `fakePlayer1.willMakeMoves(TOP_LEFT, TOP_CENTER, TOP_RIGHT)`?
Or `fakePlayer1.willPlayOn(TOP_ROW)`?